app/components/sidebars/main/channels_list/channel_item/index.js

```diff
@@ -44,10 +44,10 @@ function makeMapStateToProps() {
                 }
                 const teammateNameDisplay = getTeammateNameDisplaySetting(state);
                 displayName = displayUsername(teammate, teammateNameDisplay, false);
-                isArchived = channel.delete_at > 0;
             }
         }
 
+        isArchived = channel.delete_at > 0;
```

Comment:
@mkraft It was changed that way to show deactivated users from jumpTo search

<img width="374" alt="screen shot 2018-10-10 at 12 09 55 pm" src="https://user-images.githubusercontent.com/4973621/46717421-8b06de80-cc85-11e8-95fd-b32a2364ca90.png">

If we consider archive flag we don't show them in the UI. Can you think of a way to omit search results from being hidden by isArchived flag?

Something like instead of adding this flag to all channels avoid search channels so we can show them when searched from jumpTo?

Reply:
That is pretty messed up code because of filtered component logic for jumpTo and sidebar logic and it becomes really tricky to not make bugs around it. Sorry about that.

Reply:
@sudheerDev I'm confused about how the `isArchived` affects listing deactivated users.

Reply:
Why is the `ChannelList` component being used to display a list of users?

Reply:
>  ChannelList component being used to display a list of users?

Yes, the name does not make sense please feel free to change it. I don't see a reason why we should not use it now given that it has all the logic. Even if we use another component instead of `ChannelList` we will run into same set of issues. 

The root problem lies in the abstraction of logic in the parent components(filtered list) from what i understand.

Reply:
I’ll jump in here, the reason is the same reason as CMD+K on the webapp that you can open a DM with a user you don’t have a direct message channel just yet